manager/manager.go

```diff
@@ -324,6 +330,8 @@ func (m *Manager) Run(parent context.Context) error {
 	api.RegisterHealthServer(m.server, authenticatedHealthAPI)
 	api.RegisterRaftMembershipServer(m.server, proxyRaftMembershipAPI)
 	api.RegisterControlServer(m.server, authenticatedControlAPI)
+	api.RegisterLogsServer(m.localserver, localProxyLogsAPI)
```

Comment:
Can you please move this line below to group it with the other calls that register services on `m.localserver`?